src/convertToObject.js

```diff
@@ -12,7 +12,14 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const propertyObj = {};
+  const propertiesArray = sourceString.split(';');
+
+  for (let property of propertiesArray) {
```

Comment:
will be better to use array methods with arrow function(forEach and arrow callback)

Reply:
Done